pandas/core/groupby/ops.py

```diff
@@ -603,8 +603,7 @@ def _aggregate(
     ):
         if agg_func is libgroupby.group_nth:
             # different signature from the others
-            # TODO: should we be using min_count instead of hard-coding it?
-            agg_func(result, counts, values, comp_ids, rank=1, min_count=-1)
```

Comment:
one alternative here is to remove the keyword from the libgroupby function.

Reply:
I went with that

Actually we don't use the min_count argument in the libgroupby function - I guess that it was put there to homogenize signatures, but now they're different anyways